lib.sh

```diff
@@ -167,13 +168,21 @@ SPAMHAUS=/etc/spamhaus.wl
 ENVASIVE=/etc/apache2/mods-available/mod-evasive.load
 APACHE2=/etc/apache2/apache2.conf
 # Full text Search
-es_install() {
+opensearch_install() {
     INDEX_USER=$(gen_passwd "$SHUF" '[:lower:]')
-    ROREST=$(gen_passwd "$SHUF" "A-Za-z0-9")
+    OPNSREST=$(gen_passwd "$SHUF" "A-Za-z0-9")
     nc_fts="ark74/nc_fts"
-    fts_es_name="fts_esror"
+    opens_fts="opensearchproject/opensearch"
+    fts_node="fts_os-node"
+}
+create_certs(){
+    rm "$OPNSDIR"/opensearch_certs.sh
+    curl -s https://raw.githubusercontent.com/Ark74/vm/add_opensearch_fts_engine/static/opensearch_certs.sh > "$OPNSDIR"/opensearch_certs.sh
+    #curl -s "$STATIC"/opensearch_certs.sh > "$OPNSDIR"/opensearch_certs.sh
```

Comment:
TODO: replace when merged

Reply:
replace it now, we can fix bugs when merged. I trust that you tested it already.

Reply:
Let's merge first, then I can have a look with find/replace